examples/features/xds/README.md

```diff
@@ -0,0 +1,32 @@
+# gRPC xDS example
+
+The xDS example is a Hello World client/server capable of being configured with
+the XDS management protocol. Out-of-the-box it behaves the same as hello world.
```

Comment:
"it behaves the same as [our other hello world example](https://github.com/grpc/grpc-go/tree/master/examples/helloworld)"?